### ISSUE TITLE
Test: multiple accessibility failures

### ISSUE THREAD

**Issue by plocket** [plocket/docassemble-cucumber#777]
One of our Steps lets authors test that their pages pass automated accessibility analyzers. Make an interview with 2-3 separate pages that have accessibility issues on them. Make sure the tests get to the end and report on all the accessibility failures.

Markdown lets us add our own html, so this should be achievable.
